### ISSUE TITLE
NodeSchool and Workshopper Clones

### ISSUE THREAD

**Issue by rvagg** [nodeschool/discussions#253]
@timoxley suggested we kick of a discussion here regarding the various clones and rip-offs that are popping up and whether there's anything we're inclined to do about them. Perhaps it's all benign and *more material is good material*; but worth discussing anyway.

* [learnserversidejavascript](https://github.com/Jalalhejazi/learnserversidejavascript) is a munge of learnyounode and expressworks; exercises have been plucked from both and repackaged without attribution and with a new copyright asserted. See https://github.com/Jalalhejazi/learnserversidejavascript/issues/1

* A certain database company is working on a complete duplication of the NodeSchool concept, including similar domain name, website, artwork, even some of the workshopper names are similar. Unfortunately there is absolutely no reference to NodeSchool.io or any of the workshoppers that seem to have inspired the naming. I'm not going to go in to details because it's pre-launch and perhaps these things will be resolved. Also I haven't had a chance to engage with them on the specifics so it's worth giving them the benefit of the doubt. Email me if you really want details.

Useful items to start discussion though. How much of this matters? How much does everyone care? At what point would a third party's co-opting of the brand start to be cause for concern? What steps should we take, if any, to protect the brand, material, etc.?

Perhaps @mikeal can form a foundation for us, he loves foundations.

Also, report other clones here for discussion.

**Comment by martinheidegger** [comment]
This should be mentioned somewhere.

**Comment by Sequoia** [comment]
Is this an ongoing issue? I bumped the thread from @Jalalhejazi but in the end unless someone wants to initiate legal action he's gonna do what he's gonna do, & it doesn't look like he's doing much with it anyway.

Was the artwork license question answered? I don't see anything on nodeschool.io

If there's no additional action steps here I vote close.

**Comment by martinheidegger** [comment]
I opened a license issue at the nodeschool homepage.

Also, I think @mikeal 's recommendation is sound. Should I open issues with all workshoppers?

**Comment by Sequoia** [comment]
@martinheidegger better you open pull requests on all workshoppers :smile_cat:.  If the individual workshopper authors aren't experiencing problems with their content (and it is their content) they probably don't need/want to be told to fix their license.

otoh if they have shared/borrowed content (artwork) that's mislicensed, yeah, but again I think enforcement should be left to the respective authors' discretion unless they've *asked* for help.

**Comment by martinheidegger** [comment]
How about to close this issue after the PR's are created :smile: 

(help welcome)

**Comment by Sequoia** [comment]
* [ ] Create license recommendations for workshopper's (somewhere)

Ultimately authors will choose how they wish to license their own works, unless there's GPL on workshopper itself in which case they must also license GPL & include the license in their package. *I* like GPL but that's for the workshopper krew to decide :smile_cat:

**Comment by Sequoia** [comment]
@rvagg @timoxley @mikeal et. al: so @martinheidegger & I took this issue & ran with it (see also #930). The ultimate conclusion was (as I understand it) that since workshopper lessons are created by disparate individuals on volunteer time & since the point of MIT license is to allow derivers & adders-on to use whatever the heck license they want, it didn't make sense to have a "recommended" license. Instead, a PR has been added to the `nodeschool/workshoppers/readme.md` informing people that they can pick what license they will for *their* lesson & directing them to choosealicense.com if they need help.

Does this sufficiently address the licensing question in your views, more or less? I consider this "resolved" but want to check in with yall as the ticket creators. :+1:
